master/internal/experiment.go

```diff
@@ -446,6 +453,7 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 	case *apiv1.PauseExperimentRequest:
 		switch ok := e.updateState(ctx, model.PausedState); ok {
 		case true:
+			ctx.Tell(e.hpImportance, hpimportance.ExperimentPaused{ID: e.ID})
```

Comment:
comment: if someone needs to upgrade a cluster, they're going to pause a lot of experiments and shut it down in some cases, just so that's taken into consideration. as i see it shouldnt matter.0

Reply:
Hmm... It'll matter when the master restarts once I have it scan for all the stuff that was in-flight and trigger all those jobs... Maybe we should only do this when a specific experiment is manually paused, or we have some other filter for these mass events... I'll think about this.

Reply:
So what we decided was not to automatically trigger stuff when an experiment is paused at all, but to immediately implement the ability to manually request specific metrics at artbitary times anyway. There's a lot of value on the latter feature anyway, and it eliminates any real need for the former. So we'll just avoid the pausing-storms entirely. Still need to implement the ability to restart in-progress jobs after a restart but there's a separate JIRA for that.